src/pages/components/utils/customCluster/customCluster.tsx

```diff
@@ -0,0 +1,69 @@
+import L, { DivIcon, MarkerCluster } from 'leaflet'
+
+const createClusterCustomIcon = (cluster: MarkerCluster): DivIcon => {
+  const childCount = cluster.getChildCount()
+  const imgPath = getIconForGroup(childCount)
+  const className = getClassNameForGroup(childCount)
+
+  const innerHtml = `
+    <div class="marker-container">
+      <span>${childCount}</span>
+       <img src="${imgPath}" alt="haha" />
+    </div>`
+
+  return new DivIcon({
+    html: innerHtml,
+    className: className,
+    iconSize: new L.Point(40, 40),
+  })
+}
+export default createClusterCustomIcon
+
+const getIconForGroup = (groupSize: number): string => {
+  const imgPath = process.env.PUBLIC_URL + '/bus-groups/'
+
+  type groupsToImg = {
+    readonly [index: number]: string
+  }
+
+  const groups: groupsToImg = {
+    2: '2.svg',
+    3: '3.svg',
+    10: '10.svg',
+    30: '30.svg',
+  }
+
+  let icon = ''
+
+  for (const i of Object.keys(groups)) {
```

Comment:
even if relying on the insertion order will work, it's still not a common practice - let's try to avoid it
https://stackoverflow.com/questions/5525795/does-javascript-guarantee-object-property-order

maybe something like 
```js
[2,3,10,30].findIndex(groupSize => groupSize >= numOfNearbyBusses)
```
will do the trick better (?)